ynput/ayon-core
# Nuke: fixing multiple intermediate file streams publishing

## Changelog Description
- Update `colorspace` assignment to use a method
- Remove redundant line adding custom tags - it was blocking Extract review transcoding.

## Testing notes:
1. create multiple presets in `ayon+settings://nuke/publish/ExtractReviewIntermediates/outputs`
2. publish files and notice that multiple mp4 were created.
